github: added PULL_REQEUEST_TEMPLATE for PR maintenance

leemgs opened the pull request:
This commit is to improve PR template (e.g., summary, goal, description, reproduction, and so on) that can be submitted by contributers to evolve the FauxPilot software.

Signed-off-by: Geunsik Lim <leemgs@gmail.com>
Signed-off-by: Geunsik Lim <geunsik.lim@samsung.com>

fdegier commented on .github/pull-request-template.md:
```diff
@@ -0,0 +1,21 @@
+*
+
+## 2. Related JIRA Tasks & GitHub Issues
+* https://code.sec.samsung.net/jira/browse/{JIRA}
```
Since this is not related to Samsung I suggest to remove it.
```suggestion
```

leemgs replied:
> Since this is not related to Samsung I suggest to remove it.

Nice Catch. Updated.

fdegier commented on .github/PULL_REQUEST_TEMPLATE.md:
```diff
@@ -0,0 +1,39 @@
+
+---
+# [Template] PR Description
```
Why are there 2 PR templates? The second one seems more appropriate for FauxPilot. What do you think?

leemgs commented on .github/pull-request-template.md:
```diff
@@ -0,0 +1,21 @@
+* {links to design docs, confluence pages, etc}
+*
+
+## 2. Related JIRA Tasks & GitHub Issues
```
> 

Thanks. Fixed.

leemgs commented on .github/pull-request-template.md:
```diff
@@ -0,0 +1,21 @@
+## 3. How to write a commit title for changelog
+Commit title must be started with one of the eleven types listed below.
+e.g., build, chore, ci, docs, feat, fix, perf, refactor, revert, style, and test.
+* https://github.sec.samsung.net/RS-SA/code-review-bot/blob/develop/Documentation/changelog-commit-rule.md
```
> Url is not publicly accessible.

Fixed. Thanks.

leemgs commented:
To reviewers

**Changelog**
 * Version 2
   - Applied comments of reviewer(s)
   - Updated the sections of a PR writing guide for readability.

 * Version 1
   - Added Pull Request template


@moyix , @fdegier, I have updated the existing PR according to the @fdegier 's comments. Could you review the latest version?

leemgs commented:
## Screenshot after a latest version. 

![image](https://user-images.githubusercontent.com/82404/210200615-a0342697-458b-41b6-a4e4-daf6cfc62187.png)

leemgs commented:
This PR is ready to be merged. Pleast review the latest version.

fdegier commented:
Thank you @leemgs 🙏
